libglusterfs/src/common-utils.c

```diff
@@ -5485,3 +5486,10 @@ get_xattrs_to_heal()
 {
     return xattrs_to_heal;
 }
+
+void
+gf_buf_cleanup(gf_buf_t *buf)
```

Comment:
I think it would be better to define this as `static inline` in the `.h`. This will allow better optimizations by the compiler.